packages/vite/src/generators/configuration/configuration.ts

```diff
@@ -198,7 +198,34 @@ export async function viteConfigurationGenerator(
     });
   }
 
-  createOrEditViteConfig(tree, schema, false, projectAlreadyHasViteTargets);
+  if (!schema.skipViteConfig) {
```

Comment:
We added this new flag so that it doesn't create a vite config here? We can still create a base config which `@nx/react` overrides.

Reply:
I removed the option, and made the create-or-update handle merging the config better.